inc/dynamic-css/related-posts.php

```diff
@@ -55,9 +55,16 @@ function astra_related_posts_css( $dynamic_css ) {
 		$related_posts_section_title_line_height    = astra_get_option( 'related-posts-section-title-line-height' );
 		$related_posts_section_title_text_transform = astra_get_option( 'related-posts-section-title-text-transform' );
 
+		// Setting up container BG color by default to Related Posts's section BG color.
+		$content_bg_obj     = astra_get_option( 'content-bg-obj-responsive' );
+		$container_bg_color = '#ffffff';
+		if ( isset( $content_bg_obj['desktop'] ) && isset( $content_bg_obj['desktop']['background-color'] ) && '' !== $content_bg_obj['desktop']['background-color'] ) {
```

Comment:
```suggestion
		if ( isset( $content_bg_obj['desktop']['background-color'] ) && '' !== $content_bg_obj['desktop']['background-color'] ) {
```
Please verify this once. Direct checking isset for key to the child key should work.

Reply:
Done 👍